src/core/components/Root/index.js

```diff
@@ -0,0 +1,25 @@
+/* @flow */
+import * as React from 'react';
+import { ConnectedRouter } from 'connected-react-router';
+import { Provider } from 'react-redux';
+
+import I18nProvider from 'core/i18n/Provider';
+import type { I18nType } from 'core/types/i18n';
+import type { ReactRouterHistoryType } from 'core/types/router';
+
+type Props = {|
+  children: React.Node,
+  history: ReactRouterHistoryType,
+  i18n: I18nType,
+  store: Object,
```

Comment:
We have a rudimentary [store type](https://github.com/mozilla/addons-frontend/blob/d7e6609988c0f2a079e7e88d36b086d7c56c79d2/src/core/types/redux.js#L9) you can use.